docs/tools_user_guide.md

```diff
@@ -293,61 +293,44 @@ You can see the options available for any command by running `benchit COMMAND --
 
-### Input Script
+### Input Scripts
 
-Name of the script (.py) file, within the search directory, to be benchmarked.
+Name of one or more script (.py) files to be benchmarked.
 
-Usage: 
-- `benchit INPUT_SCRIPT` 
-- `benchit benchmark INPUT_SCRIPT`
+Examples: 
+- `benchit models/selftest/linear.py` 
+- `benchit models/selftest/linear.py models/selftest/twolayer.py` 
 
-You can leverage model hashes (see [Model Hashes](#model-hashes)) to filter which models in a script will be acted on, in the following manner:
-  - `benchit benchmark example.py::hash_0` will only benchmark the model corresponding to `hash_0`.
-  - You can also supply multiple hashes, for example `benchit benchmark example.py::hash_0,hash_1` will benchmark the models corresponding to both `hash_0` and `hash_1`.
+The `INPUT_SCRIPTS` argument is also available as an API argument:
+- `benchmark_script(input_scripts=...)`
 
-Available as an API argument:
-- `benchmark_script(input_script=...)`
-
-> See the [Filtering Model Hashes tutorial](https://github.com/groq/mlagility/blob/main/examples/cli/discovery.md#filtering-model-hashes) for a detailed example.
-
-### Search Directory
-
-Path to a directory (defaults to the command line command line location), which serves as the search path for input scripts.
-
-Usage:
-- `benchit benchmark INPUT_SCRIPT --search-dir SEARCH_DIR`
-  - Benchmark script INPUT_SCIPT which can be found in directory SEARCH_DIR.
-- `benchit benchmark --search-dir SEARCH_DIR --all`
-  - Benchmark all scripts which can be found in directory SEARCH_DIR.
+You may also use [Bash regular expressions](https://tldp.org/LDP/Bash-Beginners-Guide/html/sect_04_01.html) to locate the scripts you want to benchmark.
 
-Available as an API argument:
-- `benchmark_script(search_dir=...)`
+Examples:
+- `benchit *.py`
+  - Benchmark all scripts which can be found at the command line location.
+- `benchit models/*/*.py`
+  - Benchmark all mlagility scripts.
 
-> See the [Search Directory tutorial](https://github.com/groq/mlagility/blob/main/examples/cli/discovery.md#search-directory) for a detailed example.
+> See the [Benchmark Multiple Scripts tutorial](https://github.com/groq/mlagility/blob/main/examples/cli/discovery.md#benchmark-multiple-scripts) for a detailed example.
 
-### Benchmark All Scripts
+You can also leverage model hashes (see [Model Hashes](#model-hashes)) to filter which models in a script will be acted on, in the following manner:
+  - `benchit example.py::hash_0` will only benchmark the model corresponding to `hash_0`.
+  - You can also supply multiple hashes, for example `benchit example.py::hash_0,hash_1` will benchmark the models corresponding to both `hash_0` and `hash_1`.
 
-Benchmark all models within all script (.py) files in the search directory.
+_Note_: Using bash regular expressions and filtering model by hashes are mutually exclusive. To filter models by hashes, please provide the full path of the Python script rather than a regular expression. 
```

Comment:
```suggestion
> _Note_: Using bash regular expressions and filtering model by hashes are mutually exclusive. To filter models by hashes, please provide the full path of the Python script rather than a regular expression. 
```

I've been putting all the notes in quote blocks

Reply:
Also don't say "please" for something that is truly mutually exclusive.

Reply:
Modified!